src/styles/main.scss

```diff
@@ -0,0 +1,54 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.clock {
+  position: relative;
+  height: 80vmin;
+  width: 80vmin;
+  border-radius: 100%;
+  border: 6px dotted black;
+  margin: auto;
+  margin-top: 50px;
+
+  &__center {
+  height: 30px;
+  width: 30px;
+  background-color: #FFA500;
+  border-radius: 100%;
+  position: relative;
+  margin: auto;
+  top: 50%;
+  }
```

Comment:
```suggestion
  &__center {
    height: 30px;
    width: 30px;
    background-color: #FFA500;
    border-radius: 100%;
    position: relative;
    margin: auto;
    top: 50%;
  }
```

Reply:
`margin: auto` works only for horizontal margins

Reply:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order